css/style.css

```diff
@@ -0,0 +1,827 @@
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  position: fixed;
+  z-index: 2;
+  top: 0;
+  background-color: #fefefe;
+}
+
+.mid-div-left {
+  display: none;
+}
+
+.top-elements {
+  display: none;
+}
+
+.tnw-logo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  width: 106px;
+}
+
+.tnw-logo img {
+  width: 76px;
+  max-height: 28px;
+}
+
+.right-elements {
+  display: flex;
+  max-width: 100px;
+  border: 0.1px solid rgb(190, 188, 188);
+  text-align: center;
+  align-items: center;
+  justify-content: center;
```

Comment:
[REQUIRED] these stylings are repeating more than 5 times. we can create a custom class for this styling and add as required. We have to remove all the repeated codes.